iDiff/cmd/iDiff.go

```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
 	"os"
 	"testing/runtimes-common/iDiff/differs"
+	"testing/runtimes-common/iDiff/utils"
```

Comment:
any reason this is all in the top level `testing` package? could it just be in `runtimes-common/iDiff`?